topics/icrc_25_signer_interaction_standard.md

```diff
@@ -143,9 +144,109 @@ Request
     "method": "icrc25_request_permission",
     "params": {
         "version": "1",
-        "scopes": [{
-          "method": "icrc25_canister_call"
-        }],
+        "scopes": [
+            {
+                "method": "icrc25_get_identities"
+            },
+            {
+                "method": "icrc25_canister_call"
+            }
+        ]
+    }
+}
+```
+
+Response
+```json
+{
+    "id": 1,
+    "jsonrpc": "2.0",
+    "result": {
+        "version": "1",
+        "scopes": [
+            {
+                "method": "icrc25_get_identities"
+            }
+        ]
+    }
+}
+```
+
+### `icrc25_get_identities`
+
+The purpose of the `icrc25_get_identities` message is for the relying party to receive information about the identities managed by the signer.
```

Comment:
It seems a bit confusing that the message and the scope use identical label `icrc25_get_identities`.  Is that intentional?

Reply:
Yes, this is intentional. See the explanation in the `scopes` section:
> A scope is the permission to invoke specific JSON-RPC 2.0 method on the signer. A scope is identified by the `method` property which matches the `method` name of the JSON-RPC 2.0 call it relates to.